build/features.py

```diff
@@ -942,29 +943,34 @@ def description(self):
         return "Optimization and Tuning"
 
     def enabled(self, build):
-        build.flags['optimize'] = util.get_flags(build.env, 'optimize', 1)
-        if int(build.flags['optimize']):
+        build.flags['optimize'] = SCons.ARGUMENTS.get('optimize', '')
+        if build.flags['optimize'] is not 'portable':
```

Comment:
something like 'off' shouldn't trigger an assertion. 'disable' is not intuitive for people coming from other build sytems